.eslintrc

```diff
@@ -20,31 +20,40 @@
     "camelcase": "off",
     "class-methods-use-this": "off",
     "consistent-return": "off",
+    "import/no-cycle": "off",
```

Comment:
Do we still need to disable this? I resolved some issues and merged them recently, this is the commit:
https://github.com/acl-services/paprika/pull/80/commits/5c4397d155cbdd17508e315665a1b80e11a09061

And a little more context:
https://github.com/acl-services/paprika/pull/80#pullrequestreview-241432722

Reply:
yes, I still have problems with it (just test it), I will resolve the conversation.